src/workerd/jsg/resource.h

```diff
@@ -191,6 +194,75 @@ template <typename TypeWrapper,
     typename Indexes>
 struct MethodCallback;
 
+// Specialization for const methods that take `Lock&` as their first parameter.
```

Comment:
I previously suggested removing the new specialization for const methods, you resolved my comment, but it seems like it's still here? Why do we need this?